django-backend/fecfiler/transactions/migrations/0020_trigger_save_on_transactions.py

```diff
@@ -4,71 +4,72 @@
 logger = structlog.get_logger(__name__)
 
 
-class Migration(migrations.Migration):
-
-    dependencies = [
-        ("transactions", "0019_aggregate_committee_controls"),
-    ]
+def trigger_save_on_transactions(apps, schema_editor):
+    transactions = apps.get_model("transactions", "transaction")
+    committees = apps.get_model("committee_accounts", "committeeaccount")
+    contacts = apps.get_model("contacts", "contact")
 
-    def trigger_save_on_transactions(apps, schema_editor):
```

Comment:
I was wondering how this worked but now it makes sense. We're never trying to call it from an instance of Migration like `migration.trigger_save_on_transaction()` or `self.trigger_save_on_transaction()`.  So it's being defined in a class where it SHOULD be a class method, but it's never being used as one, so the missing `self` is not an issue